src/services/services.tsx

```diff
@@ -0,0 +1,13 @@
+const getData:any = async (setData:any) => {
+    await fetch('https://jsonplaceholder.typicode.com/posts/1')
+        .then(response => response.json())
+        .then(response => {
+            setData(response)
+            console.log(response)
+        })
+        .catch((err) => {
+            console.log(err)
+        })
+}
```

Comment:
1. forgot to return something
2. don't mix `async-await` with `then` - use `async-await` only

Reply:
need descuss